package.json

```diff
@@ -54,6 +54,7 @@
     "remap-istanbul": "^0.7.0",
     "sinon": "^1.17.4",
     "sinon-chai": "^2.8.0",
+    "ts-node": "^8.3.0",
```

Comment:
This file and the package-lock.json are not required and should be removed from this PR.